configuration/azure/appconfig/appconfig.go

```diff
@@ -295,15 +295,21 @@ func (r *ConfigurationStore) Subscribe(ctx context.Context, req *configuration.S
 }
 
 func (r *ConfigurationStore) doSubscribe(ctx context.Context, req *configuration.SubscribeRequest, handler configuration.UpdateHandler, sentinelKey string, id string) {
+	var etagVal *azcore.ETag
```

Comment:
This is always an empty Etag value... doesn't that mean you keep getting updates all the time?

It seems you should be storing the etag somewhere and be using this in subsequent calls using this `OnlyIfChanged` option.

Reply:
cool, somehow missed that even though I was looking for it.